br/tests/br_full_ddl/run.sh

```diff
@@ -110,6 +111,20 @@ fi
 # clear restore environment
 run_sql "DROP DATABASE $DB;"
 
+# restore full
+echo "restore start..."
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/restore/restore-createtables-error=return(true)"
+run_br restore full -s "local://$TEST_DIR/$DB" --pd $PD_ADDR --log-file $RESTORE_LOG --ddl-batch-size=128 || { cat $RESTORE_LOG; }
+export GO_FAILPOINTS=""
+
+pause_count=$(cat $RESTORE_LOG | grep "panic"| wc -l | xargs)
```

Comment:
maybe `panic_count` instead of `pause_count`?

(Also, the `xargs` may be removed?)

Reply:
agreed.